apps/etterlatte-saksbehandling-ui/client/src/components/person/behandlingsliste.tsx

```diff
@@ -65,34 +50,50 @@ const lenkeTilBehandling = (behandlingSammendrag: IBehandlingsammendrag): string
 }
 
 export const Behandlingsliste = ({ behandlinger }: { behandlinger: IBehandlingsammendrag[] }) => {
+  const sorterteBehandlinger = behandlinger.sort((a, b) =>
+    new Date(b.behandlingOpprettet!) > new Date(a.behandlingOpprettet!) ? 1 : -1
+  )
+
+  /**
+   * TODO:
+   *  Burde ha en form for paginering av behandlinger. Visning av tabellen og henting av vedtakssammendrag henger
+   *  seg opp når det er for mange elementer i listen. Eksempelvis testbruker STOR SNERK som har 210 behandlinger i
```

Comment:
Vil den da også gjøre tilsvarende mange kall mot vedtakssammendrag?

Reply:
Japp, så ikke helt heldig at alle vises samtidig. Ikke et stort problem riktig enda, men må fikses om ikke alt for lenge.